client/post-editor/editor-term-selector/index.jsx

```diff
@@ -77,7 +80,7 @@ class EditorTermSelector extends Component {
 export default connect(
 	( state ) => {
 		const siteId = getSelectedSiteId( state );
-		const postId = getEditorPostId( state );
+		const postId = getEditorPostId( state ) || '';
```

Comment:
I seem to think this is needed, but this PR - rather the one before it and this one - have been a long strange trip - I'll try it out.